services/backend/services/v3/config/providers.json

```diff
@@ -8,5 +8,27 @@
     "successRedirect": "/auth/account",
     "failureRedirect": "/local",
     "failureFlash": true
+  },
+  "ldap": {
+    "provider": "ldap",
+    "authScheme": "ldap",
+    "module": "passport-ldapauth",
+    "authPath": "/auth/msad",
+    "successRedirect": "/auth/account",
+    "failureRedirect": "/msad",
+    "session": true,
+    "json": true,
+    "failureFlash": true,
+    "profileAttributesFromLDAP": {
+      "displayName": "displayName",
+      "email": "mail"
+    },
+    "server": {
+      "url": "ldap://ldap:389",
+      "bindDn": "cn=admin,dc=facility",
+      "bindCredentials": "admin",
+      "searchBase": "ou=users,dc=facility",
+      "searchFilter": "(uid={{username}})"
+    }
```

Comment:
I completely agree. Now that we have features toggles, this should all be combinable. I was also thinking, that maybe  one could, instead of mounting the whole config folder in the `fe`, just mount the files which are needed for that specific config and then use the include.path trick, as we do in the be (so there will be a compose.b3.yaml in the fe which has the volume mount of config.v3.json). Maybe this way it will be clearer for the user what it's used? I don't have a strong opinion though...